ajax/libs/leaflet-groupedlayercontrol/package.json

```diff
@@ -0,0 +1,64 @@
+{
+  "name": "leaflet-groupedlayercontrol",
+  "filename" : "leaflet.groupedlayercontrol.min.js",
+  "version": "0.6.0",
+  "description": "A Leaflet layer control with support for grouping overlays together",
+  "main": "dist/leaflet.groupedlayercontrol.min.js",
```

Comment:
@smendez93 Let's remove the information we don't need. `"main"`, `"directories"`, `"scripts"`, `"bugs"`, `"devDependencies"`, `"dependencies"` and `"jspm"`. Thanks.